spec/index.html

```diff
@@ -556,28 +556,28 @@ <h3>Triples</h3>
       |p| is called the <dfn>predicate</dfn> of the triple, and
       |o| is called the <dfn>object</dfn> of the triple.</p>
 
-    <p>A <dfn data-local-lt="quoted">quoted graph</dfn> is an <a>RDF graph</a> that is
-      used as the <a>subject</a> or <a>object</a> of triple.</p>
+    <p>A <dfn>graph term</dfn> is an <a>RDF graph</a> that is
+      used as the <a>subject</a> or <a>object</a> of a <a>triple</a>.</p>
 
     <p class="note">By the given definitions,
       the <a>subject</a> of an <a>RDF triple</a> may be either an <a>IRI</a>,
-      a <a>blank node</a>, or a <a>quoted graph</a>;
+      a <a>blank node</a>, or a <a>graph term</a>;
       the <a>predicate</a> of an <a>RDF triple</a> may only be an <a>IRI</a>;
       the <a>object</a> of an <a>RDF triple</a> may be either an <a>IRI</a>,
-      a <a>blank node</a>, a <a>literal</a> or a <a>quoted graph</a>.</p>
+      a <a>blank node</a>, a <a>literal</a> or a <a>graph term</a>.</p>
 
-    <p class="note">The definition of <a>quoted graph</a> is recursive.
-      That is, a <a>quoted graph</a> can itself have a
+    <p class="note">The definition of <a>graph term</a> is recursive.
+      That is, a <a>graph term</a> can itself contain
       triples where the <a>subject</a> or <a>object</a> components
-      are <a>quoted graphs</a>.
-      However, by this definition, cycles of <a>quoted graphs</a> cannot be created.</p>
+      are <a>graph terms</a>.
```

Comment:
You'll need to manually apply this, as it encompasses lines that were already deleted (572 & 573), and only makes sense if changing both the line preceding (571) and the line following (572) the deletion.
```suggestion
      triples where the <a>subject</a> or <a>object</a> component
      is a <a>graph term</a>.
```

Reply:
Hm. It occurs to me, that I see no reason why *both* subject and object couldn't be graph terms. There might be domain/range restrictions that limit the predicates usable between graph terms, but that seems reasonable to me. Thus, this revised suggestion —
```suggestion
      triples where either or both the <a>subject</a> and/or <a>object</a> component
      is a <a>graph term</a>.
```

Reply:
Yes - it's quite important. We already have illustrative syntax cases for "forward" and "backward" links on graph terms.

It is the graph terms themselves that can't be recursive (i.e. no cycles via graph terms inside graph terms).

Reply:
@TallTed I re-created this change suggestion directly in the current version of the PR; see: https://github.com/w3c/rdf-concepts/pull/67/files#r1361896571